### PR TITLE
Dx improvements

### PR DESCRIPTION
**Pull request by greg0ire**
Right now it looks hard to contribute to this library.

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Spea) [inline phpunit.xml]: Could you rename this file to `phpunit.xml.dist` and put `phpunit.xml` in `.gitignore` please?

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@
 /docs/vendor
 .idea
 .*.swp
+/composer.lock
```
Comment (Spea) [inline .gitignore]: You can actually remove the `/`

**Review by Spea — COMMENTED** [review]
Thank you for your contribution, I only have a small request for changes :)

File: .gitignore (hunk 2 above)
Comment (greg0ire) [reply .gitignore]: That will match any `composer.lock` file no matter where it is in the directory tree, are you sure?

Comment (Spea) [reply .gitignore]: Yes, as `vendor` is in gitignore anyway and we don't have any other `composer.lock` in this project, so it is fine.

Comment (greg0ire) [reply .gitignore]: If you're sure, you're sure, I pushed a new version :)

File: phpdoc.xml.dist [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Spea) [inline phpdoc.xml.dist]: You accidentally used php**doc**.xml.dist instead of php**unit**.xml.dist :)

Comment (greg0ire) [reply phpdoc.xml.dist]: Oh damn, silly me, good catch!

**Comment by Spea** [comment]
Thank you!

**Comment by greg0ire** [comment]
Thank you for the quick review, you're a great maintainer!

**Pull request merged**